adapters/syncer_test.go

```diff
@@ -0,0 +1 @@
+package adapters
```

Comment:
Got an empty file here too... not sure if I only just noticed this now, or it became empty in those last changes.

Reply:
gotcha, ok. no worries